include/fbgemm/FbgemmI8DepthwiseAvx2.h

```diff
@@ -7,6 +7,7 @@
 #pragma once
 
 #include <cstdint>
+#include <stdexcept>  // needed for Microsoft VC 2019 Enterprise version on windows-latest
```

Comment:
Actually I don't see any dependency to stdexcept in this file. Can you share compiler error message? Maybe the dependency is actually in one of the other header/source files including this. We want to minimize header includes from platform dependent files (*Avx2.{cc,h} or *Avx512.{cc,h}).

Reply:
Here is the error. I created a paste.
https://our.internmc.facebook.com/intern/paste/P125233423/

Reply:
Got it. In these case, please include <stdexcept> from FbgemmI8Depthwise3x3Avx2.cc (and other files if needed). We want to minimize the number of headers included from a header file, unless code in header file actually uses the header file.

Reply:
I moved stdexcept.h to cc files.